pkg/state/fee_validation_test.go

```diff
@@ -271,3 +272,46 @@ func TestSponsorshipFeeReduction(t *testing.T) {
 	require.NoError(t, checkMinFeeWaves(sponsorshipA, params, false, maxEstimatorVersion))
 	require.NoError(t, checkMinFeeWaves(sponsorshipB, params, false, maxEstimatorVersion))
 }
+
+func randomScript(size uint64) (proto.Script, error) {
+	var s proto.Script = make([]byte, size)
+	_, err := rand.Read(s[:])
+	if err != nil {
+		return nil, err
+	}
+	return s, nil
+}
+
```

Comment:
Please, add test on zero bytes script, it's a valid use case, setting empty script removes script from account. According to scala implementations setting zero size script should
cost 1 fee unit.